src/parsers/manifestjson.js

```diff
@@ -203,4 +207,19 @@ export default class ManifestJSONParser {
       version: this.parsedJSON.version,
     };
   }
+
+  _checkForDuplicateKeys() {
+    try {
+      RJSON.parse(this._jsonString, { duplicate: true, warnings: true });
+    } catch (err) {
+      if (err.message.includes('Duplicate key')) {
+        let key = err.message.match(/Duplicate key: (.*)/)[1];
```

Comment:
If there's more errors after the duplicate key one, what do you end up with `key` representing?